src/main/java/org/cactoos/proc/ProcOf.java

```diff
@@ -70,7 +53,7 @@ public ProcOf(final Func<X, ?> fnc) {
      * Ctor.
      * @param prc The proc
      */
-    private ProcOf(final Proc<X> prc) {
+    public ProcOf(final Proc<X> prc) {
```

Comment:
@victornoel This type of ctor doesn't make much sense, why would one want to wrap pre-existing `Proc` with `ProcOf`?

Reply:
@andreoss it's mainly useful to force type a lambda: `new ProcOf(x -> { /* do something with x */ ; })`